bin/bs_ios

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# bs_ios uploads app binaries for UI testing on BrowserStack.
+#
+# This version of the script works only with the V2 endpoint (XC Test Plans).
+# This script assumes that Xcode Test Plan name is TestPlan.xctestplan.
+#
+# It forwards all arguments to "patrol build ios", so you can pass --target,
+# --flavor, --exclude etc. just as you would pass them to "patrol_cli".
+#
+# It can also be configured with the following environment variables:
+BROWSERSTACK_CREDS="${BROWSERSTACK_CREDS:-}"
+BROWSERSTACK_PROJECT="${BROWSERSTACK_PROJECT:-}"
+BROWSERSTACK_IOS_DEVICES="${BROWSERSTACK_IOS_DEVICES:-}"
+BROWSERSTACK_SKIP_BUILD="${BROWSERSTACK_SKIP_BUILD:-0}"
+BROWSERSTACK_IDLE_TIMEOUT="${BROWSERSTACK_IDLE_TIMEOUT:-900}"
```

Comment:
You can create a script in your project, let's call it `bs_envvars`, that exports these variables, something like:

```
#!/usr/bin/env bash
set -euo pipefail

export BROWSERSTACK_PROJECT=HeadphonesApp
export BROWSERSTACK_IDLE_TIMEOUT=420
```

and then you [source](https://superuser.com/questions/46139/what-does-source-do) that script: `. ./bs_envvars`. This will set these values for the current shell.

Reply:
hm, I will forget to source it more times than I will remember to do it

Reply:
I'm not aware of a better solution that is equally simple

Reply:
fix: don't forget xD